spec/factories/doorkeeper_applications.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :doorkeeper_application, class: Doorkeeper::Application do
+    name "foo"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.